packages/config/src/config/build.js

```diff
@@ -57,6 +57,7 @@ export default () => ({
   plugins: [],
   terser: {},
   hardSource: false,
+  aggressiveClientServerCodeRemoval: false,
```

Comment:
How about aggressiveCodeRemoval ?

Reply:
I thought it'd be too unspecific, but I'm fine with that :relaxed:

Reply:
A doc would cover this, I just thought the shorter name will be better for usage.